github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRLabelExistsEvent/config.groovy

```diff
@@ -2,7 +2,7 @@ package org.jenkinsci.plugins.github.pullrequest.events.impl.GitHubPRLabelExists
 
 def f = namespace(lib.FormTagLib);
 
-f.entry(field: "skip", title: "Skip PR?") {
+f.entry(field: "skip", title: "Skip pull requests with label(s)?") {
```

Comment:
Let me remember, all event triggering logic allows you combine events and do PR skips. I.e. you have "not ready for merge" label. So all events naming is strict and logical (i hope).